spec/queries/get_content_collection_spec.rb

```diff
@@ -0,0 +1,63 @@
+require "rails_helper"
+
+RSpec.describe Queries::GetContentCollection do
+  it "returns the content items of the given format" do
+    create(:draft_content_item, base_path: '/a', format: 'topic')
+    create(:draft_content_item, base_path: '/b',  format: 'topic')
+    create(:draft_content_item, base_path: '/c',  format: 'mainstream_browse_page')
+
+    expect(Queries::GetContentCollection.new(
+      format: 'topic',
+      fields: ['base_path'],
+    ).call).to eq([
+      { "base_path" => "/a", "publication_state" => "draft" },
+      { "base_path" => "/b", "publication_state" => "draft" },
+    ])
+  end
```

Comment:
It might make sense to include the content_id in the response as well since we're keying content items by them. What do you think?